pages/agent/v3/elastic_ci_aws.md.erb

```diff
@@ -104,10 +106,10 @@ If you want to store your secrets unencrypted, you can disable it entirely with
 
 <!-- vale off -->
 
-* [Amazon Linux 2 LTS](https://aws.amazon.com/amazon-linux-2/)
```

Comment:
I dropped the LTS suffix from the Elastic CI Stack in-repo docs as I can’t see Amazon refer to it as such anywhere.

Reply:
:100: